supertokens_python/recipe/emailverification/recipe.py

```diff
@@ -450,20 +452,23 @@ def __init__(
         claim: EmailVerificationClaimClass,
         ev_claim_validators: EmailVerificationClaimValidators,
         refetch_time_on_false_in_seconds: int,
-        max_age_in_seconds: int,
+        max_age_in_seconds: Optional[int] = None,
```

Comment:
This looks fine, but I'm unsure why this was marked optional if we assert that this isn't `None` in `shouldRefetch`